lib/local_authority/use_case/get_return.rb

```
@@ -1,20 +1,23 @@
 # frozen_string_literal: true
 
 class LocalAuthority::UseCase::GetReturn
-  def initialize(return_gateway:)
+  def initialize(return_gateway:,return_update_gateway:)
     @return_gateway = return_gateway
+    @return_update_gateway = return_update_gateway
   end
 
   def execute(id:)
     found_return = @return_gateway.find_by(id: id)
+    updates = @return_update_gateway.updates_for(return_id: id)
+
     if found_return.nil?
       {}
```

silent failure is a bit confusing - are we going to put some magic in here later?